infra/multi_cluster_service.tf

```diff
@@ -14,6 +14,14 @@
  * limitations under the License.
  */
 
+resource "google_project_iam_member" "my_service_account_role_network_viewer" {
+  project = var.project_id
+  role    = "roles/compute.networkViewer"
+  member  = "serviceAccount:${google_service_account.my_service_account.email}"
+  depends_on = [
+    module.enable_google_apis
+  ]
+}
```

Comment:
This is necessary for multi-cluster Services, according to [cloud.google.com](https://cloud.google.com/kubernetes-engine/docs/how-to/multi-cluster-services#authenticating).